### ISSUE TITLE
Gold income

### ISSUE THREAD

**Issue by Profputricide** [Heishe/idleraiders-second-run#438]
The gold income really feels low when you're farming endgame. Currently having to idle for more than a week just to afford a reset.
Maybe make higher map monsters drop bonus gold? (like add 1 gold value when you start from map 1 copy again or something)
Or make raids actually drop decent amount of gold, as i remember being told that raids were supposed to be farmed for gold. (currently the time it takes to run them, makes them worth less gold than staying in a farm map)

**Comment by TravisGesslein** [comment]
Yes, gold/xp and artifact tuning definitely needs to be worked on. Undoubtedly it's messy right now. Before release, farm maps will be rebalanced a little bit, and that would probably a good time to adjust all the currency gain in the game.
